app/base/static/assets/js/vis/simple-bar-chart.js

```diff
@@ -4,8 +4,8 @@ class SimpleBarChart {
     CHART_HEIGHT = 600;
     */
 
-    CHART_WIDTH = document.getElementById('charts').offsetWidth-50;
-    CHART_HEIGHT = window.innerHeight-120;
+    CHART_WIDTH = document.getElementById('charts').offsetWidth - 50;
```

Comment:
Let's aim to avoid using magic numbers and define them in the common.js.

Reply:
create a static number called as  static MainContent_Gap = 220;
the recalculation is:

    ContentMarginTop = 10px
    ContentPaddingTop = 30px
    ContentPadingBottom = 30px
    CardMarginTop = 30px     
    CardMarginBottom = 30px
    CardHeadPaddingTop = 15px
    CardHeadPaddingBottom = 15px
    IcondHeight = 53px
so total = 213px
then let the number be 220